docs/service-introspection-onboarding.md

```diff
@@ -25,6 +45,115 @@ following properties of this storage before proceeding:
 Once you have a storage account with a table, you may proceed to start updating
 the pipelines to send data to Spektate storage.
 
+**Note:** The Azure storage account is needed to store information about your
+pipelines and services that is displayed by service introspection.
+
+2. Create a table. Follow these
+   [instructions](https://docs.microsoft.com/en-us/azure/storage/tables/table-storage-quickstart-portal).
+
+### Pipelines Configuration
+
+The Bedrock GitOps pipelines need to be configured to start sending data to
+`spk` service introspection. This is done by adding a script snippet in each
+`azure-pipelines.yml` configuration.
+
+#### Configure a variable group
+
+To send data from Azure pipelines to the service introspection storage created
+previously a variable group needs to be configured in Azure DevOps (where the
+pipelines are).
+
+To configure the variable group run: `spk variable-group create`
+
+#### Pipeline 1: CI pipeline configuration
+
+The CI pipeline runs from the source repository to build a docker image.
+
+Paste the following task in its corresponding `azure-pipelines.yml`:
+
+```yaml
+- bash: |
+    curl $SCRIPT > script.sh
+    chmod +x ./script.sh
+    tag_name="hello-spektate-$(Build.SourceBranchName)-$(Build.BuildId)"
+    commitId=$(Build.SourceVersion)
+    commitId=$(echo "${commitId:0:7}")
+    ./script.sh $(ACCOUNT_NAME) $(ACCOUNT_KEY) $(TABLE_NAME) $(PARTITION_KEY) p1 $(Build.BuildId) imageTag $tag_name commitId $commitId service $(Build.Repository.Name)
+  displayName: Update manifest pipeline details in CJ db
+  env:
+    SCRIPT: https://raw.githubusercontent.com/catalystcode/spk/master/scripts/update_introspection.sh
+```
+
+This task will update the service introspection storage table for every build
+that runs from the source repository. This information will be available for use
+by service introspection.
+
+##### Note:
+
+- Make sure the variable `tag_name` is set to the tag name for the image being
+  built in your docker step.
+
+- Add the task early on so that information is sent to spk.
+
+- Add the task before the crucial steps in your pipeline. This will capture
+  details about failures if the important steps fail.
+
+#### Pipeline 2: CD release pipeline (ACR to HLD) configuration
+
+The CD release pipeline updates the docker image number in the HLD.
+
```

Comment:
Need a step for the use to add a variable group reference like  below. 
Make sure that you update the pipelines in the following steps to include this variable group, such as below:

variables:
  - group: <your-variable-group-name from above>